.vscode/launch.json

```diff
@@ -21,7 +21,7 @@
             "runtimeExecutable": "${execPath}",
             "args": [
                 "--extensionDevelopmentPath=${workspaceFolder}",
-                "--extensionTestsPath=${workspaceFolder}/out/src/test"
+                "--extensionTestsPath=${workspaceFolder}/out/src/test/index.js"
```

Comment:
Why add `.../index.js`? https://code.visualstudio.com/api/working-with-extensions/testing-extension just ends with `.../index`

Reply:
To disambiguate file vs folder